x/superfluid/keeper/intermediary_account.go

```diff
@@ -100,9 +100,6 @@ func (k Keeper) GetOrCreateIntermediaryAccount(ctx sdk.Context, denom, valAddr s
 	// create a new account. We use base accounts, as this is what's done for cosmwasm smart contract accounts.
 	// and in the off-chance someone manages to find a bug that forces the account's creation.
 	if !k.ak.HasAccount(ctx, intermediaryAcct.GetAccAddress()) {
-		// UNFORKING v2 TODO: I now need to set NextAccountNumber with k.ak.NextAccountNumber(ctx) instead of using zero, due to new invariant checks.
```

Comment:
This is correct, and we will need to update the `in-place-testnet` for v27 based on this invariant :+1:

Reply:
https://linear.app/osmosis/issue/CHAIN-621/fix-in-place-testnet-for-v26

created, good shout :bow: